chef/data_bags/crowbar/migrate/nova_dashboard/026_use_keystone_session_timeout.rb

```diff
@@ -0,0 +1,13 @@
+def upgrade ta, td, a, d
+  keystone_instance = a['keystone_instance']
+  proposal_obj = ProposalObject.find_proposal("keystone", keystone_instance)
+  keystone_timeout = (proposal_obj["attributes"]["keystone"]["token_expiration"] || 3600)/60
```

Comment:
Hrm, why 3600 and not 14400 as in https://github.com/crowbar/barclamp-keystone/blob/release/tex/master/chef/data_bags/crowbar/bc-template-keystone.json#L25 ?